plugins/notifications-backend-module-email-notification-processor/README.md

```diff
@@ -0,0 +1,23 @@
+# @backstage/plugin-notifications-backend-module-email-notification-processor
+
+The email-notification-processor backend module for the notifications plugin.
+The idea is to send the notification details via email when the notification severity is above the configured threshold.
+
+Example configuration:
+
+```yaml
+backend:
```

Comment:
Should live under `notifications` instead `backend` config

Reply:
why not backend? notifications is for the backend plugin only.

Reply:
I moved it to the top level like other backend plugins. E.g. linguist

Reply:
fixed